### PR TITLE
Add SurfaceID fragment

### LINKED ISSUE #197: Ability to map surface order to panels on pull
When reading panels, the order of the panel in the `gem` file is important to the allocation of other data in workflows utilising File_Toolkit. It would be good to know on the panel which order in the `gem` file they were when they were pulled.

FYI @rboulton-BH @FraserGreenroyd

### PR DESCRIPTION
**Pull request by kprusicka**
### Issues addressed by this PR
Fixes #197 

 ### Test files

 ### Changelog

 ### Additional comments

### CODE REVIEW

**Comment by kprusicka** [comment]
@BHoMBot check compliance

File: IES_oM/Fragments/SurfaceIndexFragment.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+using System.Threading.Tasks;
+using BH.oM.Base;
+using System.ComponentModel;
+
+namespace BH.oM.IES.Fragments
+{
+    [Description("Fragment containing the surface index from IES")]
+    public class SurfaceIndexFragment : IFragment
+    {
+        public virtual int SurfaceID { get; set; } = -1;
```
Comment (FraserGreenroyd) [inline IES_oM/Fragments/SurfaceIndexFragment.cs:14]: Please add a description for the property with an example usage

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]
Please address the compliance issues too

**Comment by FraserGreenroyd** [comment]
@BHoMBot fix copyright headers ref. 9094972189

**Comment by FraserGreenroyd** [comment]
@BHoMBot fix project file ref. 9094960875

**Comment by FraserGreenroyd** [comment]
@BHoMBot check compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot check compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot check required

**Review by FraserGreenroyd — APPROVED** [review]
Changes made, all looks good to me, great work @kprusicka

**Comment by FraserGreenroyd** [comment]
@BHoMBot check ready-to-merge

**Pull request merged**